config/application.rb

```diff
@@ -151,6 +151,10 @@ class Application < Rails::Application
       config.lookbook.auto_refresh = false
       config.lookbook.project_name = "#{APP_NAME} Component Previews"
       config.lookbook.ui_theme = 'blue'
+      if Identity::Hostdata.env != 'dev'
+        # so we can embed a lookbook component into the dev docs
+        config.lookbook.preview_embeds.policy = "ALLOWALL"
```

Comment:
@aduth   so I've got good news and bad news...

good news:
The Lookbook docs have a config param specifically for making the components embeddable that we can set in application.rb

bad news:
It only supports the X-Frame parameters so we are limited to "ALLOWALL" or 'SAMEORIGIN" since [ALLOW-FROM `origin` is deprecated](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Frame-Options#allow-from_origin)

basically nothing was working when I added my previous code to the preview component, because it was always getting overwritten by the X-Frame-Options SAMEORIGIN being set by Lookbook so I think this is really the only way to get it to work